sql_grader/datasets/rating.sql

```diff
@@ -0,0 +1,47 @@
+/*
+https://s3-us-west-2.amazonaws.com/prod-c2g/db/Winter2013/files/rating.sql
+*/
+
+/* Delete the tables if they already exist */
+drop table if exists Movie;
+drop table if exists Reviewer;
+drop table if exists Rating;
+
+/* Create the schema for our tables */
+create table Movie(mID int, title text, year int, director text);
+create table Reviewer(rID int, name text);
+create table Rating(rID int, mID int, stars int, ratingDate date);
+
+/* Populate the tables with our data */
+insert into Movie values(101, 'Gone with the Wind', 1939, 'Victor Fleming');
+insert into Movie values(102, 'Star Wars', 1977, 'George Lucas');
+insert into Movie values(103, 'The Sound of Music', 1965, 'Robert Wise');
+insert into Movie values(104, 'E.T.', 1982, 'Steven Spielberg');
+insert into Movie values(105, 'Titanic', 1997, 'James Cameron');
+insert into Movie values(106, 'Snow White', 1937, null);
+insert into Movie values(107, 'Avatar', 2009, 'James Cameron');
+insert into Movie values(108, 'Raiders of the Lost Ark', 1981, 'Steven Spielberg');
```

Comment:
FWIW, if you run into performance issues (which seems unlikely), you might want to try reducing the number of SQL statements so you have one "insert" per table, and then a bunch of values, e.g.:

```
insert into Movie (mID, title, year, director)
    values
        (101, 'Gone with the Wind', 1939, 'Victor Fleming'),
        (102, 'Star Wars', 1977, 'George Lucas'),
        ...
```